content/en/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/index.md

```diff
@@ -398,180 +388,31 @@ to hold the configuration of the NGINX server:
     EOF
     {{< /text >}}
 
-1.  Define a `ServiceEntry` and a `VirtualService` for `nginx.example.com` to instruct Istio to direct traffic destined
-    to `nginx.example.com` to your NGINX server:
-
-    {{< text bash >}}
-    $ kubectl apply -f - <<EOF
-    apiVersion: networking.istio.io/v1alpha3
-    kind: ServiceEntry
-    metadata:
-      name: nginx
-    spec:
-      hosts:
-      - nginx.example.com
-      ports:
-      - number: 80
-        name: http
-        protocol: HTTP
-      - number: 443
-        name: https
-        protocol: HTTPS
-      resolution: DNS
-      endpoints:
-      - address: my-nginx.mesh-external.svc.cluster.local
-        ports:
-          https: 443
-    ---
-    apiVersion: networking.istio.io/v1alpha3
-    kind: VirtualService
-    metadata:
-      name: nginx
-    spec:
-      hosts:
-      - nginx.example.com
-      tls:
-      - match:
-        - port: 443
-          sniHosts:
-          - nginx.example.com
-        route:
-        - destination:
-            host: nginx.example.com
-            port:
-              number: 443
-          weight: 100
-    EOF
-    {{< /text >}}
-
-#### Deploy a container to test the NGINX deployment
-
-1.  Create Kubernetes [Secrets](https://kubernetes.io/docs/concepts/configuration/secret/) to hold the client's and CA
-    certificates:
+1.  Deploy the [sleep]({{< github_tree >}}/samples/sleep) sample to test sending requests to the NGINX server:
```

Comment:
Why did we delete the sleep sample at the end of part 1, when we need to restart it again here?

I suggest deleting this step and also removing the sleep sample from the above part 1 cleanup section.

Reply:
This part is to deploy sleep with client certificates to test the NGINX mutual TLS server.

Reply:
Now without that cert mounted sleep, this deployment is the same as used in TLS mode originally. I would remove it from cleanup and remove this deployment

Reply:
I see, the part that @nschhina deleted. See my comment, above. Even so, shouldn't we use the sleep pod without certs for the later test? Currently, that one's deleted up in the part 1 cleanup. If we want a source pod with certs, just for the nginx test, shouldn't we just create a second one for that (in a different ns)?

Reply:
I am fine with removing the cert-mounted sleep and the test to make the document shorter.